### ISSUE TITLE
Framework7-Svelte Popover bug

### ISSUE THREAD

**Issue by Applemann** [framework7io/framework7#3574]
* Framework7 version: 5.5.0
* Platform and Target: Android 9 Cordova app
* Live Link or JSFiddle: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_. You can start from forking the following JSFiddle template: [JSFiddle](https://jsfiddle.net/nolimits4web/o2ejupu1/)

### Describe the bug
When I have popover in Navbar and go throw it into some page so popover is closed, but some
invisible instance of popover is still there.

### To Reproduce
1. Download project from this repository: https://github.com/Applemann/F7PopoverBug
2. Run it in Android 9
3. Open menu in right up corner
4. Go to 'Nastaveni'
5. Click to last (Nocni rezim) Toggle (now it works)
6. Go back to the first page
7. Click to 'Go second page' button
8. Open menu in right up corner
9. Go to 'Nastaveni'
10. Click to last (Nocni rezim) Toggle (now don't works)

### Expected behavior
Normal working as in first page.

### Actual Behavior
ListButton in popover don't close the popover because some shadow instance is still there.

### Video
[F7PopupBug-video.webm.zip](https://github.com/framework7io/framework7/files/4328563/F7PopupBug-video.webm.zip)

**Comment by AliMamed** [comment]
You may reproduce this bug in Chrome also.
I think this is similar to #3581 as this happens when you have multiple popups (which are same as popups are extended class of modal).
I also have problems with it - I will try to propose a PR.
Meanwhile in your particular case you may fix it by replacing line 5 in your `Menu.svelte` with the following: 
```
<ListButton popoverClose=".{name}.modal-in" href="/Settings" title="{$_('menu.settings')}" />
```

**Comment by AliMamed** [comment]
@nolimits4web could you please look at it? Or give me idea of the lifecycle.
To reproduce you could simply download the project and run it with `npm start`. And repeat Appleman's steps.
A workaround I gave looks to work fine. And I was looking to change `close` method in `utils/modals-methods.js` to filter if `$el.length > 1` by `modal-in` class name, but I just get the popover stuck with `modal-out` class then.
I am sure this issue of the same nature as #3581 and I vote to fix it as I have the same problem in my React project where I have popover in navbar.
Thanks in advance.

**Comment by nolimits4web** [comment]
Fixed in ba16a210896ed8d28661f4661da9a94be6709f4a
